src/cloud/live/sens-data-over.md

```diff
@@ -60,6 +62,17 @@ All system configurations are set during build and deploy according to the follo
 1. If an environment variable does not exist, use the configuration from a `MAGENTO_CLOUD_RELATIONSHIPS` name-value pair in the [`.magento.app.yaml` file][app-yaml]. Ignore the default configuration.
 1. If an environment variable does not exist and `MAGENTO_CLOUD_RELATIONSHIPS` does not contain a name-value pair, remove all customized configuration and use the values from the default configuration.
 
+If the same setting is configured in multiple places, Magento relies on the following configuration hierarchy to determine which value to apply to the environment :
+
+|Priority|Configuration<br/>Method|Description|
+|---|---|---|
+|1 | Project Web UI<br>environment variables | Values added from the _Variables_ tab of environment configuration in the Project Web UI. We recommend specifying values here for sensitive or environment-specific configurations. Settings specified here cannot be edited from the Magento Admin. See [Environment configuration variables][].|
+|2 | `.magento.app.yaml` | Values added in the `variables` section of the `.magento.app.yaml` file. We recommend specifying values here to ensure consistent configuration across all environments. **Do not specify sensitive values in the `.magento.app.yaml` file.** See [Application settings][app-yaml].|
+|3 | `app/etc/env.php` | Environment-specific configuration values stored here are added by using the Magento `app:config:dump` command. Set the system-specific and sensitive values using environment variables or the Magento CLI. See [Sensitive data](#sensitive-data). The `env.php` file is **not** included in source control.|
+|4 | `app/etc/config.php` | Values stored here are added by using the Magento `app:config:dump` command. Shared configuration values are added to `config.php`. Set shared configuration from the Magento Admin or using the Magento CLI. See [Consistent configuration](#consistent-configuration). The `config.php` file is included in source control.|
+|5 | Database | Values stored here are added by setting configurations in the Magento Admin. Note that configurations set using any of the preceding methods are locked (greyed out) and cannot be edited from the Magento Admin.|
+|6 | `config.xml` | Many configurations have default values set in a the `config.xml` file for a module. If Magento cannot find any value set by any of the preceding methods, it falls back to the default value, if set.|
+
```

Comment:
I would also mention .magento.env.yaml as data from some configurations (e.g. service configurations) will override data in env.php

Reply:
Agreed but that is separate from this. This is a table for Magento application configuration hierarchy. This page is about "Configuration management for store settings" not service configurations. I had a long conversation with @meker12 last night and .magento.env.yaml is a different type of configuration (service & platform).

Reply:
@BarnyShergold agree. This is about system configurations only. Probably I was confused by `MAGENTO_CLOUD_RELATIONSHIPS` mentioned above

Reply:
Yes - I think this whole page needs a rework, especially after your comments earlier. I'm going to raise an issue so we look at it. 👍